meson_options.txt

```diff
@@ -223,6 +223,14 @@ option('plugin_logitech_bulkcontroller',
     'false': 'disabled',
   },
 )
+option('plugin_logitech_rallysystem',
+  type: 'feature',
+  description: 'Logitech Rally System support',
+  deprecated: {
```

Comment:
Don't add deprecated options for new code.

Reply:
I removed this -- I don't thikn it needs to be configurable at all.